src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,53 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+
+public enum Rank {
+
+    FIRST(6, 200000000),
+    SECOND(5, 30000000),
+    THIRD(5, 1500000),
+    FOURTH(4, 50000),
+    FIFTH(3, 5000),
+    NOTHING(0,0);
+
```

Comment:
불필요한 줄 바꿈은 제거해주세요! 😃 
또한 숫자의 경우 아래와 같이 3개의 숫자마다 `_`를 넣어서 가독성을 높여줄 수 있어요! 🤗 
```suggestion
    FIRST(6, 200_000_000),
    SECOND(5, 30_000_000),
    THIRD(5, 1_500_000),
    FOURTH(4, 50_000),
    FIFTH(3, 5_000),
    NOTHING(0, 0);
```